src/me/ryanhamshire/PopulationDensity/EntityEventHandler.java

```diff
@@ -124,19 +116,19 @@ public void onItemDespawn (ItemDespawnEvent event)
 		
 		//get info about the dropped item
 		ItemStack item = ((Item)entity).getItemStack();
-		
+
 		//only care about saplings
-		if(item.getType() != Material.SAPLING) return;
-		
+		if((item.getType() != Material.OAK_SAPLING) || (item.getType() != Material.SPRUCE_SAPLING) || (item.getType() != Material.BIRCH_SAPLING) || (item.getType() != Material.JUNGLE_SAPLING) || (item.getType() != Material.ACACIA_SAPLING) || (item.getType() != Material.DARK_OAK_SAPLING)) return;
```

Comment:
At this point, wouldn't it be better to check if `item.getType()` isn't in an array? https://stackoverflow.com/q/1128723/5083247

Or, maybe a switch with empty cases? For example : 

```javascript
switch (item.getType()) {
	case Material.OAK_SAPLING:
	case Material.SPRUCE_SAPLING:
	case Material.BIRCH_SAPLING:
	case Material.JUNGLE_SAPLING:
	case Material.ACACIA_SAPLING:
	case Material.DARK_OAK_SAPLING:
		return;
}
```

https://stackoverflow.com/a/6705977/5083247